cpucounters.cpp

```diff
@@ -946,17 +949,25 @@ bool PCM::discoverSystemTopology()
     }
     std::cerr << std::endl;
 #endif // PCM_DEBUG_TOPOLOGY
-#elif defined(__FreeBSD__) 
+#elif defined(__FreeBSD__) || defined(__DragonFly__)
 
     size_t size = sizeof(num_cores);
-    cpuctl_cpuid_args_t cpuid_args_freebds;
+    cpuctl_cpuid_args_t cpuid_args_freebsd;
     int fd;
 
+#if defined(__DragonFly__)
```

Comment:
FreeBSD does have kern.mp.cpus, but I think we should use hw.ncpu instead so that the two BSD variants are equivalent.